src/styles/blocks/card.scss

```diff
@@ -0,0 +1,105 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  box-sizing: border-box;
+
+  width: 200px;
+  padding: 32px 0 16px;
+  margin-right: 48px;
+  margin-bottom: 46px;
+
+  border: 1px solid $color-close-to-white;
+  border-radius: 5px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n+4) {
+    margin-bottom: 0;
+  }
+
+  &__image {
+    @include size-y-x(134px, 160px);
+
+    padding: 0 19px;
+    margin-bottom: 40px;
+  }
+
+  &__info {
+    margin-bottom: 4px;
+    padding: 0 16px;
+
+    @include fonts-parametrs($font-size-s, 500, 18px, $color-close-to-black);
+  }
+
+  &__producte-code {
+    margin-bottom: 16px;
+    padding: 0 16px;
+
+    @include fonts-parametrs($font-size-xs, 400, 14px, $color-close-to-grey);
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin-bottom: 24px;
+    padding: 0 16px;
+  }
+
+  &__review {
+    @include fonts-parametrs($font-size-xs, 400, 14px, $color-close-to-black);
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+
+    margin-bottom: 16px;
+    padding: 0 16px;
+  }
+
+  &__text {
+    @include fonts-parametrs($font-size-s, 400, 18px, $color-close-to-grey);
+  }
+
+  &__num {
+    @include fonts-parametrs($font-size-l, 700, 18px, $color-close-to-black);
+  }
+
+  &__button {
+    padding: 0 16px;
+  }
+
+  &__link-button {
+    display: block;
+    box-sizing: border-box;
+
+    width: 100%;
+    height: 40px;
+
+    padding: 11px;
+
+    @include fonts-parametrs($font-size-m, 700, 16px, $color-white);
+
+    text-transform: uppercase;
+    text-align: center;
+    text-decoration: none;
+
+    background-color: $color-close-to-azure;
+
+    border: 1px solid $color-close-to-azure;
+    border-radius: 5px;
+  }
+
+  &__link-button:hover {
+    color: $color-close-to-azure;
+
+    transition: 0.3s;
+
+    background-color: $color-close-to-white;
+  }
+}
```

Comment:
You should use nesting, put your **:hover** inside **&__link-button**

Reply:
I agree